### DISCUSSION TITLE
Update paperless-ngx script

### DISCUSSION THREAD

**Discussion by easyas314** [community-scripts/ProxmoxVE#499] [Q&A]
(btw - what's the appropriate github discussion category for requesting changes to scripts?)

Could Tika and Gotenberg could be added as default, or as an option to paperless-ngx to support MS Office and Open Office files?

See [paperless conf doc](https://docs.paperless-ngx.com/configuration/#tika) for overview of this option.

See this [tteck discussion](https://github.com/tteck/Proxmox/discussions/2286) for more details and an example script shared by @DeepWoods

**Comment by andygrunwald** [comment]
See related discussion: https://github.com/community-scripts/ProxmoxVE/discussions/1112

**Comment by Vendo233** [comment]
Team how can I update Paperless-ngx

![image](https://github.com/user-attachments/assets/a66a2e03-7c20-4d48-a48f-a7d4b2156484)

I have tried command "update" it returns that it has been updated but it is not

![image](https://github.com/user-attachments/assets/c2ea9f29-a4a1-44c9-a7e7-461b9527b693)

**Comment by Vendo233** [comment]
when running this script I get

![image](https://github.com/user-attachments/assets/c2d150ee-d8e2-4741-b9bf-8068afc60816)

bash -c "$(wget -qLO - https://github.com/community-scripts/ProxmoxVE/raw/main/ct/paperless-ngx.sh)"
